setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="vyxal",
-    version="2.6.1",
+    version="2.7.0",
```

Comment:
a version bump should be separate from any other changes

Reply:
I figured that too, but I realised I forgot to do it before actually making the release notes so I quickly placed it here.